subrpcserver_config.go

```diff
@@ -219,21 +219,9 @@ func (s *subRPCServerConfigs) PopulateDependencies(cc *chainControl,
 				reflect.ValueOf(genInvoiceFeatures),
 			)
 
+		// RouterRPC isn't conditionally compiled and doesn't need to be
```

Comment:
can't this case be removed altogether?

Reply:
It will error in the default clause

Reply:
okay, i missed the default case before. sgtm